### PR TITLE
Update newsletter merch unit AB test

### PR DESCRIPTION
**Pull request by buck06191**
## What does this change?
Roll out newsletter merch unit to 100% of the audience and include a control variant

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [x] Yes (please indicate your plans for DCR Implementation)

Will add this AB test to DCR

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/newsletter-merch-unit-test.js [hunk 1]
```diff
@@ -2,26 +2,30 @@
 
 export const newsletterMerchUnit: ABTest = {
     id: 'NewsletterMerchUnit',
```
Comment (alexduf) [inline static/src/javascripts/projects/common/modules/experiments/tests/newsletter-merch-unit-test.js:4]: Shouldn't the ID be changed so it's easier to distinguish between the two runs?

**Review by alexduf — APPROVED** [review]
great, we'll have to check if that's long enough (but it should be!)

Thanks Josh!

**Comment by alexduf** [comment]
Confirmed the tracking works locally by opting-in
```
https://ophan.theguardian.com/img/2?viewId=someid&abTestRegister={"NewsletterMerchUnitLighthouse":{"variantName":"variant","complete":"false"}}
```

**Comment by buck06191** [comment]
Confirmed that the new variants are visible in the network tab and on ad manager.

**Pull request merged**